CNAME

```diff
@@ -1 +0,0 @@
-gci17.fossasia.org
```

Comment:
This is a required file as this site is hosted on github (gh-pages)

Reply:
Oh okay. I'll get that back!

Reply:
Done